app/controllers/upload.controller.js

```diff
@@ -444,8 +443,8 @@ exports.upload = async (req, res) => {
 
 			const uploader = bundlr.uploader.chunkedUploader;
 
-			uploader.setChunkSize(524288);
-			uploader.setBatchSize(1);
+			uploader.setChunkSize(process.env.BUNDLR_CHUNK_SIZE || 524288);
+			uploader.setBatchSize(process.env.BUNDLR_BATCH_SIZE || 1);
```

Comment:
Where are these defaults derived from?

Reply:
The one is to do one chunk at a time. I figure it may take a little more time to upload, but would result fewer problems (like when one batch fails, what happens to the others?)

The chunk size is in bytes. The default was 25MB, which I thought was too large. I can't remember where I got this, but I thought it was the smallest chunk size.